roc_jni/src/main/impl/receiver.c

```diff
@@ -54,35 +54,59 @@ static int receiver_config_unmarshal(JNIEnv* env, roc_receiver_config* config, j
     if (jobj != NULL)
         config->resampler_profile = (roc_resampler_profile) get_resampler_profile(env, jobj);
 
+    durationClass = (*env)->FindClass(env, "java/time/Duration");
+    assert(protocolClass != NULL);
```

Comment:
protocolClass -> durationClass

also you need to declare a variable for `durationClass`.

Reply:
fix